lib/ansible/modules/cloud/amazon/ecs_service.py

```diff
@@ -298,43 +297,44 @@ def jsonize(self, service):
     def delete_service(self, service, cluster=None):
         return self.ecs.delete_service(cluster=cluster, service=service)
 
+
```

Comment:
In the future, try not to make extra whitespace changes. It might be your editor settings doing this.